microsoft-edge/webview2/concepts/data_privacy.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Data and Privacy in WebView2
+description: WebView2 data collection and practices, privacy, and the 'Diagnostic data' setting.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/03/2023
+---
+# Data and Privacy in WebView2 

```

Comment:
```suggestion
# Data and privacy in WebView2
```
* trailing space trim
* per corp style guides, use sentence case for titles/headings.  Unless you want "Privacy" as a special exception like What's New or Release Notes.